docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file.md

```diff
@@ -31,162 +31,207 @@ Steps
+            },
+            data_connectors={
+                "default_runtime_data_connector_name": {
+                    "class_name": "RuntimeDataConnector",
+                    "batch_identifiers": ["default_identifier_name"],
+                },
+                "default_inferred_data_connector_name": {
+                    "class_name": "InferredAssetSqlDataConnector",
+                    "name": "whole_table",
+                },
+            }
+        )
+    },
+    store_backend_defaults=S3StoreBackendDefaults(default_bucket_name="my_default_bucket"),
     )
-    ```
+```
 
     The following example shows a Data Context configuration with a Pandas datasource and local filesystem defaults for metadata stores. Note: imports are omitted in the following examples. Note: You may add an optional root_directory parameter to set the base location for the Store Backends.
 
-    ```python
-    data_context_config = DataContextConfig(
-        datasources={
-            "my_pandas_datasource": DatasourceConfig(
-                class_name="PandasDatasource",
-                batch_kwargs_generators={
-                    "subdir_reader": {
-                        "class_name": "SubdirReaderBatchKwargsGenerator",
-                        "base_directory": "/path/to/data",
-                    }
-                },
-            )
-        },
-        store_backend_defaults=FilesystemStoreBackendDefaults(root_directory="optional/absolute/path/for/stores"),
-    )
-    ```
+```python
+from great_expectations.data_context.types.base import DataContextConfig, DatasourceConfig, FilesystemStoreBackendDefaults
+
+data_context_config = DataContextConfig(
+    datasources={
+        "pandas": DatasourceConfig(
+            class_name="Datasource",
+            execution_engine={
+                "class_name": "PandasExecutionEngine"
+            },
+            data_connectors={
+                "tripdata_monthly_configured": {
+                    "class_name": "ConfiguredAssetFilesystemDataConnector",
+                    "base_directory": "/path/to/trip_data",
+                    "assets": {
+                        "yellow": {
+                            "pattern": r"yellow_tripdata_(\d{4})-(\d{2})\.csv$",
+                            "group_names": ["year", "month"],
+                        }
+                    },
+                }
+            },
+        )
+    },
+    store_backend_defaults=FilesystemStoreBackendDefaults(root_directory="/path/to/store/location"),
+)
+```
 
     The following example shows a Data Context configuration with an SQLAlchemy datasource and two GCS buckets for metadata stores, using some custom and some default prefixes. Note that you can still substitute environment variables as in the YAML based configuration to keep sensitive credentials out of your code. ``default_bucket_name``, ``default_project_name`` sets the default value for all stores that are not specified individually.
```

Comment:
```suggestion
    The following example shows a Data Context configuration with an SQLAlchemy datasource and two GCS buckets for metadata Stores, using some custom and some default prefixes. Note that you can still substitute environment variables as in the YAML based configuration to keep sensitive credentials out of your code. `default_bucket_name`, `default_project_name` sets the default value for all stores that are not specified individually.
```

Reply:
Single back ticks suffice for denoting code elements.  Stores are a technical term, should be title cased accordingly.  (Note: Previously "Metadata Stores" was the technical term, but it has been shortened to "Stores" in a previous review leaving the format "<Type> Stores" to specifically denote a subtype of Store in order to prevent "Metadata Stores" from being mistaken as a subtype rather than the encompassing category.)